core/app/models/spree/ability.rb

```diff
@@ -32,7 +32,7 @@ def initialize(user)
       alias_action :new_action, to: :create
       alias_action :show, to: :read
       alias_action :index, :read, to: :display
-
+      alias_action :create, :update, :destroy, to: :modify
```

Comment:
@JDutil I can't discover what :manage stands for, is it exactly the same (create, update, destroy)? In that case we should :)